frontend/packages/dev-console/integration-tests/support/pages/add-flow/catalog-page.ts

```diff
@@ -95,12 +93,17 @@ export const catalogPage = {
       }
     }
   },
-  selectTemplateTypes: (type: string | catalogTypes) => {
-    cy.get(catalogPO.catalogTypeLink)
-      .contains(type)
-      .scrollIntoView()
-      .click();
```

Comment:
This command chain was the reason for some flaky tests because the commands `contains`, `scrollIntoView` and `click` waits for a link, but the navigation component was rerendered in the meantime.